include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -323,7 +336,9 @@ namespace ADIOS2Defaults
     constexpr const_str str_engine = "engine";
     constexpr const_str str_type = "type";
     constexpr const_str str_params = "parameters";
-}
+    constexpr const_str str_usesteps = "usesteps";
+    constexpr const_str str_usesstepsAttribute = "__openPMD_internal/usesSteps";
```

Comment:
use-steps and uses-steps: once with two s once with one - good to unify?

Reply:
Hm, iirc my idea at that was that the first is used as a setter (tell the backend to use steps) and the second is used as a getter (whether the backend uses steps). But I suppose that gets lost in translation somewhere hm